pkg/validate/pod.go

```diff
@@ -30,6 +31,55 @@ import (
 	"sigs.k8s.io/cri-tools/pkg/framework"
 )
 
+// expectedMetricDescriptorNames contains all expected metric descriptor names
+// based on metrics returned by kubelet with CRI-O and cadvisor on the legacy cadvisor stats provider
+// on kubernetes 1.35.
+var expectedMetricDescriptorNames = []string{
+	"container_cpu_load_average_10s",
+	"container_cpu_load_d_average_10s",
+	"container_cpu_system_seconds_total",
+	"container_cpu_usage_seconds_total",
+	"container_cpu_user_seconds_total",
+	"container_file_descriptors",
+	"container_fs_inodes_free",
+	"container_fs_inodes_total",
+	"container_fs_io_current",
+	"container_fs_io_time_seconds_total",
```

Comment:
The fs metrics are currently not implemented in cri-o. ref: https://github.com/cri-o/cri-o/pull/9344